templates/golang.dialect-sqlite3.tmpl

```diff
@@ -20,11 +21,15 @@ func (impl {{ .Receiver }}) isConstraintError(err error) (
 {{ end -}}
 
 {{- define "open" }}
-var sqlite3DriverName = "sqlite3_" + fmt.Sprint(time.Now().UnixNano())
+var sqlite3DriverName = func() string {
+	var id [32]byte
+	rand.Read(id[:])
+	return "sqlite3_" + string(id[:])
```

Comment:
I think these names may end up in some error strings or something. Can we hex encode? `fmt.Sprintf(“sqlite3_%x”, id)` maybe?

Reply:
Fixed